modules/gapi/src/api/gproto.cpp

```diff
@@ -228,9 +231,13 @@ void cv::validate_input_arg(const GRunArg& arg)
 
 void cv::validate_input_args(const GRunArgs& args)
 {
+    GAPI_LOG_DEBUG(nullptr, "Total count: " << args.size());
```

Comment:
Is it still required?

Reply:
If yes, I'd extend the text to provide some context for those who'll see that

Reply:
I would ask about logging culture: because i do not see more logging macro usage in  opencv code. Should we avoid them?

I asked because for graph "compile" stage is no performance requirement (i believe) instead  "apply", but troubleshooting info is more important here.

If we support logging idea then I intend to add more text info here. Currently it prints something like:

```
<TIME> .... <filename>   **validate_input_args - Total args count**: 3
<TIME> .... <filename>   **validate_input_args - Process index**: 0
<TIME> .... <filename>   **validate_input_args - Process index**: 1
<TIME> .... <filename>  ASSERT : <assert Info>
```

`Total count: 0` is informative too, because for hard debugging it shows that function was called but finished successfully and developer show skip it and continue finding bug in another places ( for example)

Reply:
Its ok to have here. Not sure if the general OpenCL logic culture have been ever discussed/documented, may @alalek provide some commentary on this